quickcheck-dynamic/quickcheck-dynamic.cabal

```diff
@@ -1,16 +1,16 @@
 cabal-version:   2.2
 name:            quickcheck-dynamic
-version:         1.1.0
+version:         2.0.0
```

Comment:
Could you also update the `CHANGELOG.md` file?

Reply:
Will do.